### ISSUE TITLE
Trying to add characters or gestures crashes the app.

### ISSUE THREAD

**Issue by jtgil** [aaronkollasch/jitouch#39]
Hello,

Thank you for continuing the efforts of the original devs of JiTouch. It is a shame that they didn't continue, so thank you for spending time to bring us faithful users continued support in newer macOS.

So my Mac is a MacBook Pro retina 15" late 2013, (MacBookPro11,3). And I am unable to use the app at all. I have installed and granted permissions properly. However, when I go to add gestures and characters for the trackpad, as soon as I click "add" the app crashes, (see screenshot).

I hope you have time to look at my issue, since I have been eager to use JiTouch again since it was unsupported so long ago.

<img width="663" alt="Screenshot 2022-10-08 at 14 58 48" src="https://user-images.githubusercontent.com/52628546/194708742-9793e827-8e57-4004-9729-02b9afc863da.png">

**Comment by aaronkollasch** [comment]
Hello,
Thanks for your kind words.

What version of Jitouch are you using? Can you try installing the latest beta and see if the error still happens? https://github.com/aaronkollasch/jitouch/releases/tag/v2.82-beta.3

**Comment by jtgil** [comment]
Hello,

Thanks for the quick reply! 

So the issue also appears in the beta3.

And the version I had installed before the beta is the most recent release from you.

<img width="669" alt="Screenshot 2022-10-09 at 13 38 55" src="https://user-images.githubusercontent.com/52628546/194754952-5cce44d2-e1eb-48a0-8fe8-c51095145441.png">

**Comment by aaronkollasch** [comment]
That's strange, beta3 should fix a crash related to closing the "add" dialog.

However, I'm not able to reproduce this and I don't know how I can debug prefpane crashes on a remote machine, so I'm not sure how much I can help with this error.

What macOS version are you using?

**Comment by jtgil** [comment]
Hello! 

Apologies for the delay in getting back to you.

So there has been a development since then...

When I installed both the release and beta3 versions of JiTouch, inside the prefspane, my tabs "Trackpad", "Magic Mouse" and "Characters" were empty. I found that odd considering that initially by default I remembered it came with preinstalled actions. So I clicked the button "Restore Defaults" for all the tabs, and it reset the boxes with the default actions.

<img width="671" alt="Screenshot 2022-10-15 at 00 01 12" src="https://user-images.githubusercontent.com/52628546/195950416-08a2fccc-8d5b-4fc7-99e6-7bf767b40ebd.png">


So now I can add actions, but the trackpad does not recognize the input for the gesture...🤣




I am using a MacBook Pro 11,3 (Late 2013) with OCLP and macOS Monterey 12.6

<img width="588" alt="Screenshot 2022-10-14 at 22 35 39" src="https://user-images.githubusercontent.com/52628546/195938981-112c0cae-fe96-421c-ab17-1565db3ca656.png">

**Comment by aaronkollasch** [comment]
Hmm, I guess it was an issue with your old preferences file. I will have to check what happens if those lists start empty. Do you by chance have a copy of the old preferences file that caused the error, at `~/Library/Preferences/com.jitouch.Jitouch.plist`? If you can, please send it to me at aaron@kollasch.dev (you can remove the License entry if you want)

Does the Jitouch icon show up in the menu bar for you?
Can you go to System Preferences -> Security & Privacy -> Privacy -> Accessibility and remove Jitouch permissions with the (-) key, then restart Jitouch with the command `killall Jitouch`?

**Comment by jtgil** [comment]
Hello man,

Great news!, I finally got this working. 
Apologies again for the delay in my reply, I just wanted to test a few things before replying. And also apologies for the long message ahead.

So responding to your question before I got this to work now, is yes, JiTouch icon was appearing in the menu bar correctly, and could turn on and off properly, also all features, like add, remove and other settings could be performed without crashes or issues, it was just the finger recognition part of the trackpad for activating actions the only thing that was left to fix.

Also, sorry, but I don't have my old preferences plist in hand. 

So here, is the outline of the steps that I did, in case it can help other people with issues similar to mine. 

1. I deleted the JiTouch prefspane with the app, "appcleaner", making sure all associated files got deleted.
2. Remove permissions (only step I missed actually, but it worked anyway)
3. Then restarted my MacBook. 
4. Then installed again JiTouch, (beta version, I don't know for the release).
7. Add permissions if not already there (see note on step 2)
8. Working JiTouch!
